__tests__/SearchHit-test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {SearchHit} from '../src/components/SearchHit/SearchHit';
+import { mountWithIntl, shallowWithIntl } from '../src/helpers/intl-enzyme-test-helper.js';
+import resultData from './result_data.json';
+
+function setup() {
+  const props = {
+    intl: {
+        formatDate: jest.fn(),
+        formatTime: jest.fn(),
+        formatRelative: jest.fn(),
+        formatNumber: jest.fn(),
+        formatPlural: jest.fn(),
+        formatMessage: jest.fn(),
+        formatHTMLMessage: jest.fn(),
+        now: jest.fn(),
+    },
+    selectedViewType: 'grid',
+    record: resultData
+  }
+
+  const enzymeWrapper = shallowWithIntl(<SearchHit {...props} />)
+  return {
+    props,
+    enzymeWrapper
+  }
+}
+
+describe('components', () => {
+  describe('SearchHit', () => {
+    it('should show grid when selectedViewType is grid', () => {
+
+      const { enzymeWrapper } = setup()
+      expect(enzymeWrapper.find('.grid-item').length).toBe(1);
+
+    })
+
+    it('should show list when selectedViewType is list', () => {
+
+      const { enzymeWrapper } = setup()
+      enzymeWrapper.setProps({ selectedViewType: 'list' });
+      expect(enzymeWrapper.find('.list-item').length).toBe(1);
+
+    })
```

Comment:
Maybe add another test showing that if you change back to grid mode, that it does show a grid.